shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -416,6 +416,8 @@ - (void)viewDidAppear:(BOOL)animated {
 
 - (void)viewWillDisappear:(BOOL)animated {
   TRACE_EVENT0("flutter", "viewWillDisappear");
+  [[[UIApplication sharedApplication] keyWindow] endEditing:YES];
```

Comment:
This probably belongs in a separate PR.

Reply:
hi, this is belongs other pr, i'll remove it next commit

Reply:
done